app/controllers/peeps.rb

```diff
@@ -0,0 +1,27 @@
+class Chitter < Sinatra::Base
+  get '/' do
+    erb(:index)
+  end
+
+  get '/peeps' do
+    erb(:'peeps')
+  end
+
+  get '/peeps/new' do
+    if current_user
+      erb(:'peeps/new')
+    else
+      flash.keep[:errors] = ["You need to be logged in to post a peep"]
+      redirect '/sessions/new'
+    end
+  end
+
+  post '/peeps' do
+    time = Time.now
+    peep_content = Peep.new_line(params[:content])
```

Comment:
Nice idea, but _never do this_.  You've introduced presentation-layer cruft into your raw data.  What happens when you want to render your data in a different format (e.g. XML, JSON, PDF)?